src/Mappers/RecursiveTypeMapper.php

```diff
@@ -520,7 +520,6 @@ public function mapNameToType(string $typeName): Type&NamedType
 
             return $this->mapClassToInterfaceOrType($className, null);
         }
-
-        throw CannotMapTypeException::createForName($typeName);
+        throw TypeNotFoundException::createError($typeName);
```

Comment:
An extra line before a throw statement is a good codestyle, much like a line before a return statement.